ojssqla/ojs.py

```diff
@@ -602,6 +602,7 @@ class Licenses(Base):
     id = Column(BigInteger, primary_key=True)
     name = Column(String(200))
     pretty_name = Column(String(200))
+    description = Column(String(200))
```

Comment:
Is this the only change introduced in this PR? If so, we should really start using different commits for changes and PEP8.

Reply:
Yes it is, but I'd introduced PEP8 changes earlier to try and make sense of the code. I forgot they were there and couldn't be bothered removing them. They do no harm

Reply:
I need this for the other PR to work https://bitbucket.org/ubiquitypress/glenlivet/pull-requests/170/if-license-name-ends-in-us-will-use-values/diff